task3.0/index.html

```diff
@@ -0,0 +1,405 @@
+			<p class="before">до 30.04.21</p>
+		</div>
+		</div>
+		<div class="aboutcom">
+			<p class="about">Размер модели: L <br>
+
+Рост модели: 190 cm <br>
+
+Цвет: Хаки</p>
+			<div class="comment">
+				<div class="com1">
+					<p class="p1">Кирилл</p>
+					<div class="star">
+					<span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span></div>
+					<p>
+					</p>
+				</div>
+			</div>
+		</div>
+		<div class="product-item">
+			<p class="product-title">Мужские высокие кеды с ремешком</p>
+			
+				<div class="rating">
+					<span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span>
+				</div><span>#обувь</span>
+			
+			<div class="img1">
+			<img src="img/product09.png" alt="product09"></div>
+			<div class="namesale">
+			<div class="name">
+			<p class="nder">Bershka</p>
+			<p class="www"><a href="https://www.bershka.com/ru/%D0%BC%D1%83%D0%B6%D1%87%D0%B8%D0%BD%D1%8B/%D0%BE%D0%B4%D0%B5%D0%B6%D0%B4%D0%B0/%D0%BF%D1%80%D0%BE%D1%81%D0%BC%D0%BE%D1%82%D1%80%D0%B5%D1%82%D1%8C-%D0%B2%D1%81%D0%B5/%D0%BC%D1%83%D0%B6%D1%81%D0%BA%D0%B8%D0%B5-%D0%B2%D1%8B%D1%81%D0%BE%D0%BA%D0%B8%D0%B5-%D0%BA%D0%B5%D0%B4%D1%8B-%D1%81-%D1%80%D0%B5%D0%BC%D0%B5%D1%88%D0%BA%D0%BE%D0%BC-c1010331712p102704564.html?colorId=040">Ссылка на товар</a></p>
+		</div>
+		<div class="sale">
+			<p class="product-sale">-30%</p>
+			<p class="before">до 30.03.21</p>
+		</div>
+		</div>
+		<div class="aboutcom">
+			<p class="about">Цвет:Черный <br>
+Черные высокие кеды в баскетбольном стиле. Ремешок с застежкой на липучку.</p>
+			<div class="comment">
+				<div class="com1">
+					<p class="p1">Егор</p>
+					<div class="star">
+					<span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span></div>
+					<p>
+					</p>
+				</div>
+			</div>
+		</div>
+		</div>
+		</div>
+		<div class="block4">
+			<div class="product-item">
+			<p class="product-title">Печенье с комплиментами</p>
+			
+				<div class="rating">
+					<span>-</span>
+				</div><span>#подарки</span>
+			
+			<div class="img1">
+			<img src="img/product10.png" alt="product10"></div>
+			<div class="namesale">
+			<div class="name">
+			<p class="nder">Podarok</p>
+			<p class="www"><a href="https://podarok.by/catalog/podarki/sladosti/2789/">Ссылка на товар</a></p>
+		</div>
+		<div class="sale">
+			<p class="product-sale">-5%</p>
+			<p class="before">до 08.03.21</p>
+		</div>
+		</div>
+		<div class="aboutcom">
+			<p class="about">Печение с предсказаниями-комплиментами отлично подойдут к букету цветов как с открыткой, так и без.
+			2 печенюшки упакованы в картонную коробку и в полипропиленовую пленку, для лучшей сохранности.</p>
+			<div class="comment">
+				<div class="com1">
+					
+					<p>Отзывов пока нет.
+					</p>
+				</div>
+			</div>
+		</div>
+		</div>
+		</div>
+
+
+		</div>
+		</div>
+	</div>
+	
+	</div>
+	<div class="more">
+		<button type="button1" class="bt1">Еще</button>
```

Comment:
`type="button1"` is invalid, use `type="button"` instead. Also please use specific names with full words for all classes.